impeller/tessellator/tessellator.cc

```diff
@@ -31,8 +31,14 @@ static TESSalloc alloc = {
     0                                    /* =extraVertices */
 };
 
-Tessellator::Tessellator()
-    : c_tessellator_(::tessNewTess(&alloc), &DestroyTessellator) {}
+Tessellator::Tessellator() : c_tessellator_(nullptr, &DestroyTessellator) {
+  TESSalloc alloc = kAlloc;
+  {
```

Comment:
This nested scope can be removed

Reply:
I used those so someone doesn't accidentally use `tessellator` after the move.  That's not a compilation error, but is a clang tidy error.